webpack.config.js

```diff
@@ -105,9 +104,6 @@ const config = {
             // url-loader sets mimetype if it's passed.
             // Without this it derives it from the file extension
             mimetype: "application/font-woff",
-
-            // Output below fonts directory
-            name: "../../static/fonts/[name].[ext]",
```

Comment:
I had webpack errors here, by default webpack has the output directory config, technically there shouldn't be a need in setting it per loader, unless something rather odd happened